src/components/SchedulePage/TimeSlot.js

```diff
@@ -18,18 +21,23 @@ const useStyles = makeStyles({
 	}
 });
 
-const timeFormatter = new Intl.DateTimeFormat('en-US', { timeStyle: 'short', hour: 'numeric', minute: '2-digit' });
-
 function Timeslot({ time, events }) {
 	const classes = useStyles();
+	timeFormatter.formatToParts(time);
 	return (
 		<>
 			<Divider />
 			<Box display='flex'>
 				<ScheduleRoundedIcon fontSize={'small'} className={classes.clockImage} />
 				<div>
 					<Typography component='h2'>
-						<time dateTime={time.toISOString()}>{timeFormatter.format(time)}</time>
+						<time dateTime={time.toISOString()}>
+							{timeFormatter.format(time).toString()}
+							<Box color='gray' component='span'>
+								{' '}
+								{currentTimeZoneShort.value}
```

Comment:
Oops, I didn't realize the `.value` was needed. My intention was so that `timezone_names` would just export strings. You should move the `.value` there.